src/components/filter_drawer/FilterDrawer.vue

```diff
@@ -1,133 +1,100 @@
-import {computed, onMounted, ref} from "vue";
-import {initFilter} from "@/composable/atlas-filter-handler";
+import { computed } from 'vue'
+import type { RouteLocationNormalizedLoaded, Router } from 'vue-router'
+import { useRoute, useRouter } from 'vue-router'
+import { initFilter } from '@/composable/atlas-filter-handler'
 import DivinationCardFilterHolder
-  from "@/components/filter_drawer/filters/divination_card_filters/DivinationCardFilterHolder.vue";
-import BossFilterHolder from "@/components/filter_drawer/filters/boss_filters/BossFilterHolder.vue";
-import MapFilterHolder from "@/components/filter_drawer/filters/map_filters/MapFilterHolder.vue";
-import TextFilterHolder from "@/components/filter_drawer/filters/text_filters/TextFilterHolder.vue";
-import AtlasOverlayHolder from "@/components/filter_drawer/overlays/atlas_overlays/overlayHolder.vue";
-import type {LocationQuery, RouteLocationNormalizedLoaded, Router} from "vue-router";
-import {useRoute, useRouter} from "vue-router";
-import {useFilterStore} from "@/store/FilterStore";
-import type {LooseFilters} from "@/model/looseFilters";
-import {useFilterQueryStore} from "@/store/FilterQueryStore";
-import handleUrlQueryFilters from "@/composable/url-query-filter-handler";
-import FilterToolbar from "@/components/filter_drawer/FilterToolbar.vue";
-import {useFilterDrawerStore} from "@/store/FilterDrawerStore";
+  from '@/components/filter_drawer/filters/divination_card_filters/DivinationCardFilterHolder.vue'
+import BossFilterHolder from '@/components/filter_drawer/filters/boss_filters/BossFilterHolder.vue'
+import MapFilterHolder from '@/components/filter_drawer/filters/map_filters/MapFilterHolder.vue'
+import TextFilterHolder from '@/components/filter_drawer/filters/text_filters/TextFilterHolder.vue'
+import AtlasOverlayHolder from '@/components/filter_drawer/overlays/atlas_overlays/overlayHolder.vue'
+import { useFilterStore } from '@/store/FilterStore'
+import type { LooseFilters } from '@/model/looseFilters'
+import { useFilterQueryStore } from '@/store/FilterQueryStore'
+import handleUrlQueryFilters from '@/composable/url-query-filter-handler'
+import FilterToolbar from '@/components/filter_drawer/FilterToolbar.vue'
+import { useFilterDrawerStore } from '@/store/FilterDrawerStore'
 
-const filterStore = useFilterStore();
-const filterQueryStore = useFilterQueryStore();
-const filterDrawerStore = useFilterDrawerStore();
-const route: RouteLocationNormalizedLoaded = useRoute();
-const router: Router = useRouter();
+const filterStore = useFilterStore()
+const filterQueryStore = useFilterQueryStore()
+const filterDrawerStore = useFilterDrawerStore()
+const route: RouteLocationNormalizedLoaded = useRoute()
+const router: Router = useRouter()
 
 const drawer = computed<boolean>(() => filterDrawerStore.drawer)
 
-onMounted(() => {
-  // We need to import the AtlasFilter composable, otherwise it won't trigger, even though it is subscribed to the FilterStore
+function queryFilters() {
```

Comment:
That looks way better! Thank you!

(Firsttime really working with JS/TS let alone Vue, and it shows ;-))